packages/web/src/components/TeeProgramDetail.vue

```diff
@@ -128,6 +128,18 @@
           :description="`${program[`montant du financement`]}`"
         />
       </div>
+
```

Comment:
note: I think all this `v-if` can be refactor and simply (can be done later !)

Reply:
Yes indeed - I have not much experience in Vue so I'll leave it for later.